client/header/activity-panel/panels/inbox/card.js

```diff
@@ -1,88 +1,296 @@
 /**
  * External dependencies
  */
-import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	cloneElement,
+	Component,
+	createRef,
+	Fragment,
+} from '@wordpress/element';
+import { Button, Dropdown, Modal } from '@wordpress/components';
 import PropTypes from 'prop-types';
-import Gridicon from 'gridicons';
 import VisibilitySensor from 'react-visibility-sensor';
+import moment from 'moment';
 
 /**
  * Internal dependencies
  */
-import { ActivityCard } from '../../activity-card';
 import NoteAction from './action';
 import sanitizeHTML from 'lib/sanitize-html';
 import classnames from 'classnames';
 import { recordEvent } from 'lib/tracks';
+import './style.scss';
+import { H, Section } from '@woocommerce/components';
 
 class InboxNoteCard extends Component {
 	constructor( props ) {
 		super( props );
 		this.onVisible = this.onVisible.bind( this );
 		this.hasBeenSeen = false;
+		this.state = {
+			isDismissModalOpen: false,
+			dismissType: null,
+		};
+		this.openDismissModal = this.openDismissModal.bind( this );
+		this.closeDismissModal = this.closeDismissModal.bind( this );
+		this.bodyNotificationRef = createRef();
+	}
+
+	componentDidMount() {
+		if ( this.bodyNotificationRef.current ) {
+			this.bodyNotificationRef.current.addEventListener(
+				'click',
+				( event ) => this.handleBodyClick( event, this.props )
+			);
+		}
+	}
+
+	componentWillUnmount() {
+		if ( this.bodyNotificationRef.current ) {
+			this.bodyNotificationRef.current.removeEventListener(
+				'click',
+				( event ) => this.handleBodyClick( event, this.props )
+			);
+		}
+	}
+
+	handleBodyClick( event, props ) {
+		const innerLink = event.target.href;
+		if ( innerLink ) {
+			const { note, screen } = props;
+
+			recordEvent( 'wcadmin_inbox_action_click', {
+				note_name: note.name,
+				note_title: note.title,
+				note_content_inner_link: innerLink,
+				screen,
+			} );
+		}
 	}
 
 	// Trigger a view Tracks event when the note is seen.
 	onVisible( isVisible ) {
 		if ( isVisible && ! this.hasBeenSeen ) {
-			const { note } = this.props;
+			const { note, screen } = this.props;
 
 			recordEvent( 'inbox_note_view', {
 				note_content: note.content,
 				note_name: note.name,
 				note_title: note.title,
 				note_type: note.type,
-				note_icon: note.icon,
+				screen,
 			} );
 
 			this.hasBeenSeen = true;
 		}
 	}
 
+	openDismissModal( type ) {
+		this.setState( {
+			isDismissModalOpen: true,
+			dismissType: type,
+		} );
+	}
+
+	closeDismissModal( acepted ) {
```

Comment:
typo `acepted`

Reply:
You are right @becdetat!
I made this change and added a small refactor (addressing [this suggestion](https://github.com/woocommerce/woocommerce-admin/issues/4099#issuecomment-631720064)) in the commit [2b97132](https://github.com/woocommerce/woocommerce-admin/pull/4320/commits/2b9713292890496118c1b822333d7caaa9e8826c)